common/types/mapper/thrift/history.go

```diff
@@ -1245,3 +1245,45 @@ func ToProcessingQueueStateArrayMap(t map[string][]*history.ProcessingQueueState
 	}
 	return v
 }
+
+// FromGetFailoverInfoRequest converts internal GetFailoverInfoRequest type to thrift
+func FromGetFailoverInfoRequest(t *types.GetFailoverInfoRequest) *history.GetFailoverInfoRequest {
```

Comment:
nit: I created a `thrift-tests` folder for testing thrift mappers. For newly added types, shall we also add some tests there?

Reply:
added unit tests